app/views/partials/_search_bar.html.haml

```diff
@@ -0,0 +1,24 @@
+.search-bar-wrapper
+  %i.icon-search
+  %input#search-bar-text{placeholder: "search something", type: "text", value: ""}
+
+.filters-bar
+  %ul.filter-list
+    %li.filter
+      %select
+        %option{:value => "volvo"} Volvo
```

Comment:
this is a sample select that is definitely gonna change for some more "rails-like" way of doing dropdown selects for filter... so I'm not gonna bother here